packages/core/workers/src/WorkerFarm.js

```diff
@@ -299,69 +323,24 @@ export default class WorkerFarm extends EventEmitter {
     );
   }
 
-  createReverseHandle(fn: () => mixed) {
-    let handle = new Handle();
-    this.handles.set(handle.id, fn);
+  createReverseHandle(fn: (fn: string, args: Array<mixed>) => mixed) {
+    let handle = new Handle({workerApi: this.workerApi});
+    this.handleFns.set(handle.id, fn);
+    this.handles.push(handle);
```

Comment:
maybe the handle functions should just be part of the Handle object itself rather than two maps